backend/accessibility_templates/aa-007.xml

```diff
@@ -0,0 +1,13 @@
+<AccessibilityAttributes>
+ <DialogueEnhancementAttributes>
+  <AudioAttributes>
+   <Coding href="urn:mpeg:mpeg7:cs:AudioCodingFormatCS:2001:1"/>
+   <NumOfChannels>1</NumOfChannels>
+   <MixType href="urn:mpeg:mpeg7:cs:AudioPresentationCS:2007:2"/>
+   <AudioLanguage>mi</AudioLanguage>
```

Comment:
Can't find this language code

Reply:
see [language-subtag-registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry)
````
%%
Type: language
Subtag: mi
Description: Maori
Added: 2005-10-16
%%
````

Reply:
It was not clear to me which language codes can be used because SubtitleLanguage is restricted to ISO 639-2. It may be good to use for the majority of the language fields easy identifiable language codes.

Reply:
OK, I will change to some language codes from [ISO 639-2](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes)